app/views/step_nav/_structured_data.html.erb

```diff
@@ -0,0 +1,8 @@
+<% step_image_urls = Schemas::HowTo.how_to_images.map { |id, image|
```

Comment:
Is it OK to have this logic in the view? Would it be better to make a step_nav helper and put the logic there instead? Or are helpers a bit uncool because they're included in every view?

Reply:
I think it's OK.  It'd be nice to remove this stuff from the views altogether and generate the image_urls somewhere else ([like this for example](https://stackoverflow.com/a/31681098/170864)) as you say.  We've had problems in the past of the helpers not generating the hashed asset URLs properly though, so have resorted to this while we try to figure it out.

Reply:
I'm going to give it another crack though, thanks for the prod

Reply:
Should be better now